string serial_id
geometry_msgs/Pose goal
---
nav_msgs/Path path
bool success
